modules/serverless-rn-1-19-0.adoc

```diff
@@ -0,0 +1,19 @@
+[id="serverless-rn-1-19-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.19.0
+
+[id="new-features-1-19-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.25.1.
+* {ServerlessProductName} now uses Knative Eventing 0.25.0.
+* {ServerlessProductName} now uses Kourier 0.25.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.25.1.
+* {ServerlessProductName} now uses Knative Kafka 0.25.0.
```

Comment:
I don't know how precise we want to be here but the current Serverless 1.19 product includes all commits from upstream up to 0.25.5.
Upstream: https://github.com/knative-sandbox/eventing-kafka/releases/tag/v0.25.5
Downstream: https://github.com/openshift-knative/eventing-kafka/commits/release-v0.25
So, perhaps I'd udpate this kafka version to 0.25.5

Otherwise the RN look good to me so far

Reply:
I'm curious if it makes sense to only include major.minor for the versions. We usually include individual commits on our downstream branches which make it hard to determine which exact "micro" version it is upstream.

Reply:
@nainaz any thoughts on this one? We've always used `x.xx.x` format for versions, but I'm OK with changing it if you think that's the best thing for users.

Reply:
Updated to just use the 0.25 versions, etc. Let me know if there are any issues.